CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Features
 ### Enhancements
 ### Bug Fixes
+* Fixing KNNCodecServiceTests as reported in the gh issue:[#1593] [#1601](https://github.com/opensearch-project/k-NN/pull/1601) 
```

Comment:
Typo - 1601 vs 1593.

Also, I dont think this is a Bug Fix because it does not touch runtime code - I think it should go in Infra

Reply:
agree with Jack, if we're adding it to changelog then it's more like infra change

Reply:
I would remove this from changelog only. This is not a bug and neither an infra change